index.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+require 'vendor/autoload.php';
+
+$uri = parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH);
+
+$routes = [
+    '/' => ['controller' => 'Home', 'action' => 'index'],
+    '/about' => ['controller' => 'About', 'action' => 'index'],
+    '/contact' => ['controller' => 'Contact', 'action' => 'index'],
+    '/product/(?P<id>\d+)' => ['controller' => 'Product', 'action' => 'show'],
+];
+
+$routeFlag = false;
```

Comment:
all logic that works with the routing should be incapsulated in the Router.php class. If router cannot find route -> response->json(not found)